static/app/components/sidebar/broadcastSdkUpdates.tsx

```diff
@@ -100,7 +100,6 @@ const UpdatesList = styled('div')`
 `;
 
 const Suggestions = styled('div')`
-  margin-top: ${space(1)};
```

Comment:
Can you show what this looks like with multiple notifications? I’m not sure this is safe to remove?

Reply:
Yeah there's still spacing between the updates if there are multiple under each project (which from my understanding is less frequent) 

![Screen Shot 2021-07-21 at 12 25 03 PM](https://user-images.githubusercontent.com/4830259/126547886-dd76693c-b134-4fac-b94d-70b12d3fdd36.png)

Reply:
Sorry I mean when there’s multiple projects. I’m just trying to understand why these margins needed removed

Reply:
Ah each project has a margin already so this was acting more like line-height than anything else. Should be fine to remove